pkg/kubelet/kubelet_pods.go

```diff
@@ -2146,7 +2147,12 @@ func (kl *Kubelet) convertToAPIContainerStatuses(pod *v1.Pod, podStatus *kubecon
 		}
 		if resources.Requests != nil {
 			if cStatus.Resources != nil && cStatus.Resources.CPURequest != nil {
-				resources.Requests[v1.ResourceCPU] = cStatus.Resources.CPURequest.DeepCopy()
+				// If both the allocated & actual resources are at or below MinShares, preserve the
+				// allocated value in the API to avoid confusion and simplify comparisons.
```

Comment:
😂

Reply:
Considering that `InPlacePodVerticalScaling` is about to enter the beta stage, is it worth adding an API-level warning for this scenario? Or would a warning log in the kubelet to notify users?